src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -147,8 +147,8 @@ public function __construct(ParameterBagInterface $parameterBag = null)
 
         $this->trackResources = interface_exists('Symfony\Component\Config\Resource\ResourceInterface');
         $this->setDefinition('service_container', (new Definition(ContainerInterface::class))->setSynthetic(true)->setPublic(true));
-        $this->setAlias(PsrContainerInterface::class, new Alias('service_container', false));
-        $this->setAlias(ContainerInterface::class, new Alias('service_container', false));
+        $this->setAlias(PsrContainerInterface::class, new Alias('service_container', false))->setDeprecated(true, $deprecationTemplate = 'The "%alias_id%" service alias is deprecated. You should stop using it, as it will be removed in the future. Configure it explicitly if you want to keep using it.');
```

Comment:
`The "%alias_id%" autowiring alias is deprecated. Define it explicitly in your app if you want to keep using it.`